### PR TITLE
Handle disabled excise additional codes

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-870

### What?

I have added/removed/altered:

- [x] Handle disabled excise radio options

### Why?

I am doing this because:

- This is needed for measures that are too complex to handle at the current time

image

### CODE REVIEW

**Review by jebw — APPROVED** [review]
Looks great overall - I've a couple of suggestions around how the helper is working, personally I think the rubocop complaint is legitimate but PR looks good.

File: app/helpers/excise_helper.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    t('excise_page.small_brewers_relief_hint_text_html', small_brewers_relief_link: small_brewers_relief_link) if @step.small_brewers_relief?
+    # rubocop:enable Rails/HelperInstanceVariable
+  end
+
+  def excise_link
+    link_to(
+      t('excise_page.link_text'),
+      'https://www.gov.uk/government/publications/uk-trade-tariff-excise-duties-reliefs-drawbacks-and-allowances/uk-trade-tariff-excise-duties-reliefs-drawbacks-and-allowances',
+      class: 'govuk-link',
+      target: '_blank',
```
[the remark this answers is not in the document]
Comment (willfish) [reply app/helpers/excise_helper.rb:25]: Intereresting. I'll add the noreferrer, too

File: app/helpers/excise_helper.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      excise_link: excise_link,
+    )
+
+    sanitize(hint_text, attributes: %w[href target class])
+  end
+
+  private
+
+  def small_brewers_relief_hint_text
+    # rubocop:disable Rails/HelperInstanceVariable
```
[the remark this answers is not in the document]
Comment (willfish) [reply app/helpers/excise_helper.rb:15]: Yeah, I prefer this

File: app/helpers/excise_helper.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    t('excise_page.small_brewers_relief_hint_text_html', small_brewers_relief_link: small_brewers_relief_link) if @step.small_brewers_relief?
+    # rubocop:enable Rails/HelperInstanceVariable
+  end
+
+  def excise_link
+    link_to(
+      t('excise_page.link_text'),
+      'https://www.gov.uk/government/publications/uk-trade-tariff-excise-duties-reliefs-drawbacks-and-allowances/uk-trade-tariff-excise-duties-reliefs-drawbacks-and-allowances',
+      class: 'govuk-link',
```
[the remark this answers is not in the document]
Comment (willfish) [reply app/helpers/excise_helper.rb:22]: This is set and on my testing isn't true

[the remark this answers is not in the document]
Comment (willfish) [reply app/helpers/excise_helper.rb:22]: Probably something to review in the whole application

[the remark this answers is not in the document]
Comment (willfish) [reply app/helpers/excise_helper.rb:22]: This isn't true in my local testing:

```
<a target="_blank" rel="noopener norefferer" href="https://www.gov.uk/government/publications/uk-trade-tariff-excise-duties-reliefs-drawbacks-and-allowances/uk-trade-tariff-excise-duties-reliefs-drawbacks-and-allowances">UK Trade: excise, duties, reliefs, drawbacks and allowances (opens in new browser window)</a>
```

**Pull request merged**